user.go

```diff
@@ -30,7 +29,8 @@ func login(c httpContext, db tacitDb.TacitDB) {
 	fmt.Println("Found this user from db: ", theDbUser)
 
 	pwBytes := []byte(login.Password)
-	err = bcrypt.CompareHashAndPassword([]byte(theDbUser.Password), pwBytes)
+	err = tacitDb.TacitCrypt.CompareHashAndPassword(crypt, []byte(theDbUser.Password), pwBytes)
+
```

Comment:
this should use the injected `crypt` object for calling `CompareHashAndPassword`